confidant/utils/json.py

```diff
@@ -0,0 +1,22 @@
+import logging
+import json
```

Comment:
kind of weary of having the filename be the same as the json module.

i don't think there's actually a conflict since it's not at top-level, but worth considering a change